docs/terminal/eastsale.md

```diff
@@ -4,7 +4,226 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
-!!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
\ No newline at end of file
+| Name               | ID      | Callsign             | Frequency        | Login Identifier    |
+| ------------------ | ------- | ------------------- | ---------------- | ------------------- |
+| **East Sale TMA**  | **ESA** | **East Sale Approach** | **123.300**      | **ES_APP**          |
+
+## Airspace
+
+ES TCU controls the airspace within R360. Typically, sectors R360A through R360F are active. The enroute controllers manage the airspace beneath the active R360 sectors, with YWE controlling the southern section and ELW controlling the northern section.
+Coordination with both enroute controllers is needed if online.
+
+R393 is a reserved and is activated only for special events.
+
+If operationally necessary, ES TCU may also request the activation of M301 airspace segments for military exercises up to `F450`.
+
+Release coordination with the overlying enroute controller is required.
+
+<figure markdown>
+![ESL TMA](img/esltcu.png){ width="700" }
+<figcaption>ESL TMA</figcaption>
+</figure>
+
+
+## Logon Process
+
+Given the specific nature of the East Sale TCU airspace, surrounding controllers may not be fully aware of its boundaries. Upon logging on as East Sale TCU, it is essential to announce the activation status of R360/M301.
+
+> **Example:**
+> **ES TCU** -> **Melbourne Center**: "Request activation of R360 from SFC to F210 for military operations."
+
+## Circuit Procedures
+
+### East Sale Circuit Area (ESL CIRA)
+
+The East Sale Circuit Area (ESL CIRA) is active at all times when R360A is active. The ESL CIRA will be normally released to the ES ADC.
```

Comment:
Could we change 'will normally be released to ES ADC' to 'may be released to ADC on request'?